caffe2/python/onnx/backend.py

```diff
@@ -986,6 +986,14 @@ def supports_device(cls, device_str):
             return workspace.has_gpu_support
         return False
 
+    @classmethod
+    def is_compatible(cls, model, device='CPU', **kwargs):
+        if hasattr(super(Caffe2Backend, cls), 'is_compatible') \
+           and callable(super(Caffe2Backend, cls).is_compatible):
+            if not super(Caffe2Backend, cls).is_compatible(model, device, **kwargs):
```

Comment:
You can just do 
```
return super(Caffe2Backend, cls).is_compatible(model, device, **kwargs)
```

Reply:
Yeah, we can do that we still want to enable additional check in this function :-)